### PR TITLE
[REF-6177] move models to v1 rest api - missed in prev patch

### PR DESCRIPTION
**Pull request by yakov-g**

### CODE REVIEW

File: mlcomp/tests/test_mlpiper_cmdline.py [hunk 1]
```diff
@@ -147,14 +147,17 @@ def setup_class(cls):
         # os.chdir(mlcomp_root_path)
         # subprocess.check_call("make egg", shell=True)
 
-        mlcomp_eggs = glob.glob(os.path.join(mlcomp_root_path, "dist", "*.egg"))
+
+        eggs_pattern = "*2.*.egg" if sys.version_info[0] < 3 else "*3.*.egg"
```
Comment (zohar-mizrahi) [inline mlcomp/tests/test_mlpiper_cmdline.py]: I would use `py2`/`py3` instead of `2`/`3` correspondingly

**Review by zohar-mizrahi — APPROVED** [review]

File: mlops/parallelm/mlops/mlops.py [hunk 2]
```diff
@@ -997,11 +997,10 @@ def publish_model(self, model):
 
         return model_id
 
-    def Model(self, name="", model_format=ModelFormat.UNKNOWN, description="", user_defined=""):
+    def Model(self, name="", model_format=ModelFormat.UNKNOWN, description=""):
```
Comment (DrewParallel) [inline mlops/parallelm/mlops/mlops.py:1000]: user_defined is now annotations

**Review by DrewParallel — APPROVED** [review]

File: mlops/parallelm/mlops/mlops.py (hunk 2 above)
Comment (yakov-g) [reply mlops/parallelm/mlops/mlops.py:1000]: fixed

File: mlcomp/tests/test_mlpiper_cmdline.py (hunk 1 above)
Comment (yakov-g) [reply mlcomp/tests/test_mlpiper_cmdline.py]: fixed

**Pull request merged**